components/layout/DocsLayout.js

```diff
@@ -13,6 +14,16 @@ import ArrowRight from '../icons/ArrowRight'
 import AnnouncementRemainingDays from '../campaigns/AnnouncementRamainingDays'
 import AnnouncementHero from '../campaigns/AnnoucementHero'
 
+function generateEditLink(post) {
```

Comment:
when I think about it now, to be honest, we should always point to https://github.com/asyncapi/spec/blob/master/spec/asyncapi right? as we do not keep release branches on the spec, so anyway, if someone wants to fix something in 1.0, it is not possible as only latest version can be updated. Sorry for the initial confusion.

Does that make sense?

Reply:
aah, okay. Yeah, that's what I was somewhat confused about. Thanks for clarifying it. Gonna make the changes.